test/unit_tests/controllers/payment_initiate_controller_paynow_valid_test.dart

```diff
@@ -0,0 +1,59 @@
+import 'package:flutter_test/flutter_test.dart';
+import 'package:get/get.dart';
+import 'package:mockito/mockito.dart';
+import 'package:pispapp/controllers/app/auth_controller.dart';
+import 'package:pispapp/controllers/ephemeral/payment_initiate_controller.dart';
+import 'package:pispapp/models/user.dart';
+import 'package:pispapp/repositories/auth_repository.dart';
+import 'package:pispapp/repositories/transaction_repository.dart';
+import 'package:pispapp/routes/custom_navigator.dart';
+
+class MockAuthRepository extends Mock implements AuthRepository {}
+class MockTransactionRepository extends Mock implements TransactionRepository {}
+class MockCustomNavigator extends Mock implements CustomNavigator{}
```

Comment:
I think it would be better to be consistent with the styling... I saw you put empty lines in other files between classes